Makefile

```diff
@@ -0,0 +1,14 @@
+all: fmt build
+
+fmt:
+	go fmt ...
+
+build:
+	cd capi && go build -buildmode c-archive -o libvirtblocks.a capi.go && cc -c -o capi.o capi.c
```

Comment:
Having all commands on a single line makes it hard to see what's going on, please do something like

```
cd capi && \
go build ...
```

instead.

Reply:
Done